common/test/acceptance/tests/studio/test_studio_asset.py

```diff
@@ -35,13 +35,15 @@ def test_page_with_assets_elements_load(self):
         assert self.assert_sortable_table_heading_elements_exist()
         assert self.assert_status_element_exists()
         assert self.assert_pagination_element_exists()
+        assert self.assert_search_element_exists()
 
     def assert_page_without_filter_results_elements_load(self):
         """Make sure correct elements are on page for a filter with no results."""
         assert self.assert_studio_frontend_container_exists()
         assert not self.assert_table_exists()
         assert not self.assert_sortable_table_heading_elements_exist()
         assert not self.assert_pagination_element_exists()
+        assert not self.assert_search_element_exists()
```

Comment:
This will fail if we update to show the search bar always, but for now this is the correct validation